tests/tests.rs

```diff
@@ -43,6 +43,115 @@ fn test_local_safe_func() {
     assert_eq!(result[0].contains("__strcpy_chk"), true);
 }
 
+#[test]
+fn test_struct_mapped_files() {
+    let bin = abs_path("tests/bins/test_abort");
+    let result = GdbCommand::new(&ExecType::Local(&[&bin, "A"]))
+        .mappings()
+        .run();
+    if result.is_err() {
+        assert!(false, "{}", result.err().unwrap());
+    }
+    let result = result.unwrap();
+
+    let prmap = MappedFiles::from_gdb(&result[0]);
+    if prmap.is_err() {
+        assert!(false, "{}", prmap.err().unwrap());
+    }
+    let prmap = prmap.unwrap();
+
+    assert_eq!(
```

Comment:
The test has no sense. You should check hardcode values. For example module and offset.